src/global.d.ts

```diff
@@ -1,3 +1,6 @@
 // `nanomatch` unfortunately doesn't provide it doesn't provide its own types, there are none in DefinitelyTyped either
 // such a generic declaration works for now, but should be improved in the future
 declare module "nanomatch";
+
+// added 'uuid' as a module declartion in order for other modules to import it
+declare module "uuid";
```

Comment:
Consider taking a devDependency on `@types/uuid` with `yarn add --dev @types/uuid` instead of manually declaring it — then we'll get typings at compile time to ensure we're not using it wrong!